### PR TITLE
Ask confirmation leaving rollcall with newly scanned tokens

### PR DESCRIPTION
**Pull request by dayan9265**
This PR makes that if there are new scanned tokens (not including the organizer token), there would be a confirmation window appearing to ask the user if he really wants to leave the screen

I am currently writing tests, but since we're a bit short in time I already opened the PR to get some early feedback

In my code the `SingleRollCallProps` is not really clean, but I got type error otherwise with the goBack, and I was not sure of a nice way to do it

### CODE REVIEW

**Review by MeKHell — APPROVED** [review]
LGTM !  Nice comments.

File: fe1-web/src/features/rollCall/screens/ViewSingleRollCall.tsx [hunk 1]
```diff
@@ -9,6 +10,12 @@ import { LaoParamList } from 'core/navigation/typing/LaoParamList';
 import { Hash, PublicKey } from 'core/objects';
 import STRINGS from 'resources/strings';
 
+import { ConfirmModal } from '../../../core/components';
+import backButton from '../../../core/components/BackButton';
```
Comment (Tyratox) [inline fe1-web/src/features/rollCall/screens/ViewSingleRollCall.tsx:14]:
```suggestion
import BackButton from '../../../core/components/BackButton';
```

File: fe1-web/src/features/rollCall/screens/ViewSingleRollCall.tsx [hunk 2]
```diff
@@ -90,10 +102,73 @@ const ViewSingleRollCall = () => {
...
+    attendeePopTokensStrings.length <= 1
+  ) {
+    return backButton({ padding });
+  }
+
+  // attendees are undefined but poptokens are not -> new scanned attendees
+  if (rollCall.attendees !== undefined) {
+    // attendeesPopTokens longer than rollCall.attendees -> new scanned attendees
+    if (attendeePopTokensStrings?.length <= rollCall.attendees.length) {
+      return backButton({ padding });
```
Comment (Tyratox) [inline fe1-web/src/features/rollCall/screens/ViewSingleRollCall.tsx:133]:
```suggestion
      return <BackButton padding={padding} />;
```

File: fe1-web/src/features/rollCall/screens/ViewSingleRollCall.tsx [hunk 3]
```diff
@@ -25,6 +32,11 @@ type NavigationProps = CompositeScreenProps<
   >
 >;
 
+type SingleRollCallProps = StackNavigationProp<
+  LaoEventsParamList,
+  typeof STRINGS.events_view_single_roll_call
+>;
```
Comment (Tyratox) [inline fe1-web/src/features/rollCall/screens/ViewSingleRollCall.tsx:38]: Not ideal but should be fine here :)

**Review by Tyratox — COMMENTED** [review]
Very quick review, I hope I did not miss something bad 😅

I would certainly change the BackButton to use it as a component and not a function, otherwise LGTM!

**Pull request merged**